src/types/fhir-api-types.ts

```diff
@@ -54,14 +54,14 @@ export type FhirAPIEndpoints = {
     Response: WithIdDefined<FhirResourcesByName[Name]>;
   };
 } & {
-  // PUT /<resource>/:id
-  [Name in keyof FhirResourcesByName as `PUT /v1/fhir/dstu3/${Name}/:id`]: {
+  // PUT /<resource>/:resourceId
```

Comment:
Can you leave an inline comment explaining why this can't be named "id"? (Non-blocking)